upgrade-notes/upgrade_notes.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+if ($argc < 3) {
+    echo "Error: version or upgrade-notes path is not set. Pass the version and the path as arguments.\n";
+    exit(1);
+}
+
+$version = $argv[1];
+$upgradeNotesPath = $argv[2];
+$layoutFile = "{$upgradeNotesPath}/_layout.md";
+$outputFile = "UPGRADE-{$version}.md";
+
+function concatenateFiles($prefix, $upgradeNotesPath) {
+    $content = "";
+    $files = glob("{$upgradeNotesPath}/{$prefix}*");
+    $lastIndex = count($files) - 1;
+
+    foreach ($files as $index => $file) {
+        if (is_file($file)) {
+            $content .= file_get_contents($file);
+            // Add an empty line after each file except the last one
+            if ($index != $lastIndex) {
+                $content .= "\n\n";
+            } else {
+                $content .= "\n";
+            }
+        }
+    }
+    return $content;
+}
+
+function deleteFiles($prefix, $upgradeNotesPath) {
+    $files = glob("{$upgradeNotesPath}/{$prefix}*");
+
+    foreach ($files as $file) {
+        if (is_file($file)) {
+            unlink($file);
+        }
+    }
+}
+
+if (!file_exists($layoutFile)) {
+    echo "Error: layout file does not exist.\n";
+    exit(1);
+}
+
+// If upgrade notes file for this version exists extend it, otherwise use template
+if (file_exists($outputFile)) {
+    $upgradeNotesContent = file_get_contents($outputFile);
+} else {
+    $upgradeNotesContent = file_get_contents($layoutFile);
+}
+
+$backendNotes = concatenateFiles("backend_", $upgradeNotesPath);
+$upgradeNotesContent = str_replace("<backendNotes>", $backendNotes . "<backendNotes>", $upgradeNotesContent);
```

Comment:
I suggest use comment (`<!-- backendNotes -->`) as a placeholder, so the placeholder will not be rendered in the markdown file. (same for the storefront notes). 

Also can you extract the string into the constant, so the string is not duplicated in the code?

Reply:
That is definitely better option. Changed it.